### PR TITLE
Introduce basic prometheous exporter with power and outlet on/off state reporting

### PR DESCRIPTION
**Pull request by dlsloan**

### CODE REVIEW

File: emvue-exporter.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# SPDX-License-Identifier: BSD-3-Clause
+
+import argparse
+import prometheus_client
+import time
+
+import emvue_collect
+
+class EmVueMetrics:
+    def __init__(self, port=9686, interval=60):
```
Comment (sbates130272) [inline emvue-exporter.py]: @dlsloan has this port been officially allocated for this exporter?

File: emvue_collect.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+from pyemvue.enums import Scale, Unit
+
+try:
+    with open('.user.json') as f:
+        user_auth = json.load(f)
+except FileNotFoundError:
+    sys.exit('Expected .user.json file with {"user": "...", "password": "..."}')
+
+vue = pyemvue.PyEmVue()
+vue.login(username=user_auth['user'], password=user_auth['password'],
```
Comment (sbates130272) [inline emvue_collect.py:17]: There is another login method that uses the tokens generated from a standard login. Is it better to use the token login when it works? Not sure if this is better for the API servers or not.

**Review by sbates130272 — CHANGES_REQUESTED** [review]

File: emvue_collect.py (hunk 2 above)
Comment (dlsloan) [reply emvue_collect.py:17]: The token based login appears to be for re-authentication not initial login... not sure how long it sticks arround.

File: emvue-exporter.py (hunk 1 above)
Comment (dlsloan) [reply emvue-exporter.py]: No, I just advanced the port number we use internally in libnoload by 1

**Pull request merged**